frontend/src/components/teamsAndOrgs/teams.js

```diff
@@ -318,8 +320,8 @@ export function TeamSideBar({ team, members, managers, requestedToJoin }: Object
       rows={20}
       ready={
         typeof team.teamId === 'number' &&
-        typeof organisations !== undefined &&
-        typeof pmTeams !== undefined
+        typeof organisations !== 'undefined' &&
```

Comment:
Not directly related to the pull request, but while reviewing the code, I observed that the `TeamSideBar` component doesn't need the `organisations` and `pmTeams` variables to render. Consequently, we can eliminate the usage of these two variables.

Reply:
Made the change https://github.com/hotosm/tasking-manager/commit/5ff810cd7d79ce7bee29680a61330d0001cc4ca6.

Reply:
Works for me, especially since I had to define the variables in the first place.

Reply:
@HelNershingThapa This change should ideally be in a separate PR.

Reply:
To clarify, the change of `typeof organisations !== undefined` -> `typeof organisations !== 'undefined'` should be in a separate PR or should the removal of `typeof organisations !== 'undefined'` be in a separate PR?

Reply:
@eternaltyro recommended that the changes made in https://github.com/hotosm/tasking-manager/commit/5ff810cd7d79ce7bee29680a61330d0001cc4ca6 should have been submitted as a separate PR. I agree that it would have been preferable to have it as a separate PR.  Nevertheless, given that it has already been synced and taking into account the situation, it has been decided to overlook it this time.

Reply:
I could unsync it. It isn't that difficult to do. :)